data/global.lua

```diff
@@ -205,19 +205,40 @@ function addStamina(playerId, ...)
 			delay = configManager.getNumber(configKeys.STAMINA_GREEN_DELAY) * 60 * 1000 -- Stamina Green 12 min.
 		elseif actualStamina == 2520 then
 			player:sendTextMessage(MESSAGE_STATUS, "You are no longer refilling stamina, \z
-                                                         because your stamina is already full.")
+														 because your stamina is already full.")
 			staminaBonus.eventsPz[localPlayerId] = nil
 			return false
 		end
 
 		player:setStamina(player:getStamina() + configManager.getNumber(configKeys.STAMINA_PZ_GAIN))
 		player:sendTextMessage(MESSAGE_STATUS,
-                               string.format("%i of stamina has been refilled.",
-                                             configManager.getNumber(configKeys.STAMINA_PZ_GAIN)
-                               )
-        )
+							   string.format("%i of stamina has been refilled.",
+											 configManager.getNumber(configKeys.STAMINA_PZ_GAIN)
+							   )
+		)
 		staminaBonus.eventsPz[localPlayerId] = addEvent(addStamina, delay, nil, localPlayerId, delay)
 		return true
 	end
 	return false
 end
+
+-- Function to check for duplicate keys in a table
+-- Receives the name of the table to be checked as argument
+function checkDuplicatesValues(varName)
+	-- Retrieve the table to be checked
+	local keys = _G[varName]
```

Comment:
Can you point out why the idea of iterating all the global variables? What if the list of storages are local?